Stronger/HomeReal.swift

```diff
@@ -38,7 +38,7 @@ struct HomeReal: View {
                     }
                     
 
-                    NavigationLink(destination: WorkoutHome()) {
+                    NavigationLink(destination: WorkoutHome(presentingAccount: $presentingAccount)) {
```

Comment:
Please think about the code from a UI perspective where you want to display the account button and where you might be able to not show it. Refer to some Apple apps for some examples.